spec/common/modules/testapp_manager.js

```diff
@@ -76,7 +76,7 @@ define(['globals', 'browser-base64', 'ably'], function(ablyGlobals, base64, ably
 				}).then(function (results) {
 					callback(null, results.content.toString());
 				})
-				.catch(function(err) {
+				['catch'](function(err) {
```

Comment:
If the parsing issue is just that they don't like the newline before the method call with the dot syntax, would be clearer to just move it up to the previous line, avoid having to remember the rules of ASI

Reply:
I tried using 

```
}).then(function (results) {
					callback(null, results.content.toString());
				}).catch(function(err) {
					callback(err);
				});
```
but no luck :(

Reply:
@SimonWoolf Is this preferable?

```js
}).then(function (results) {
	callback(null, results.content.toString());
})['catch'](function(err) {
	callback(err);
});
```

Reply:
ok, so they're confused by catch being a keyword. Yup, I'd prefer that latter one to the original, as then it makes it clearer that it's a property access rather than an array literal